includes/abstracts/abstract-wc-order.php

```diff
@@ -501,7 +501,7 @@ public function get_user_id( $context = 'view' ) {
 	/**
 	 * Get user. Used by orders, not other order types like refunds.
 	 *
-	 * @return WP_User|false
```

Comment:
It suppose to return also `WP_User` instance, not a problem to keep the docblock since it's only a abstract class.